src/components/Accordion/index.tsx

```diff
@@ -40,9 +39,9 @@ const Accordion: VFC<Props> = ({ title, open = true, children }) => {
         <span>{isOpen ? '-' : '+'}</span>
       </div>
       <div data-testid="accordion-content" className={`${styles.content} ${isOpen ? styles.open : styles.close}`}>
-        {children}
+          {children}
```

Comment:
why is this changed?

Reply:
I guess this is been created by `prettier`, my editor settings are different.
Hence, this make it happened.

Reply:
Can you please revert it back to how it was?

Reply:
Revereted.